lte/gateway/c/oai/tasks/mme_app/mme_config.c

```diff
@@ -795,16 +837,37 @@ int mme_config_parse_file(mme_config_t *config_pP)
     config_pP->gummei.nb = 0;
     if (setting != NULL) {
       num = config_setting_length(setting);
+      if (num != MAX_GUMMEI) {
```

Comment:
Can the number of configured Gummeis be less than MAX_GUMMEI?

Reply:
This version of code supports only one GUMMEI, hence MAX_GUMMEI is set as 1 and the condition is there that if configured GUMMEI count is not equal to 1, MME will stop execution with Assert. 

All the possible scenarios of handling more or less than the MAX_GUMMEI count has been take care in the PR#763.